### PR TITLE
Create a z/OSMF workflow covering ZWESECUR

### PR DESCRIPTION
**Pull request by melvl01**
Signed-off-by: Vlasta Melicharkova <vm635318@broadcom.net>

Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR. If you don't know all of this information when you create this PR, don't worry. You can edit this template as you're working on it.

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Necessary documentation (if appropriate) have been added / updated
- [x] DCO signoffs have been added to all commits, including this PR

#### PR type
What type of changes does your PR introduce to Zowe? _Put an `x` in the box that applies to this PR. If you're unsure about any of them, don't hesitate to ask._ 
- [ ] Bugfix 
- [x] Feature 
- [ ] Other... Please describe:

#### Relevant issues

Fixes <https://github.com/zowe/zowe-install-packaging/issues/1047>

#### Changes proposed in this PR
Create a z/OSMF workflow covering ZWESECUR
- 

#### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

#### Does this PR do something the person installing Zowe should know about?

---
* Affected function: _general area of interest_ *

---
* Description: _1 line description_ *

---
* Part: _name of customizable file involved_  *

---
_multi-line description_

#### Is there a related doc issue or Pull Request? 

Doc issue/PR number: 

#### Other information

We expect these files to be in z/OS UNIX directory, in the build output.

### CODE REVIEW

File: files/workflows/ZWEWRF02.xml [hunk 1]
```diff
@@ -0,0 +1,888 @@
...
+  PERMIT '&amp;HLQ..*.**' CLASS(DATASET) ACCESS(ALTER) ID(&amp;SYSPROG.)
+
+  SETROPTS GENERIC(DATASET) REFRESH
+
+/* show results .................................................... */
+  LISTGRP &amp;HLQ.
+  LISTDSD PREFIX(&amp;HLQ.) ALL
+
+/* ................................................................. */
+/* only the last RC is returned, this comment ensures it is a 0      */
```
Comment (OnnoVdT) [inline files/workflows/ZWEWRF02.xml]: also incorporate issue #1044 by replacing this line with
```
/* only the last RC is returned, this command ensures it is a 0      */
PROFILE
```

File: files/workflows/ZWEWRF02.xml [hunk 2]
```diff
@@ -0,0 +1,888 @@
...
+/*                                                                   */
+/* TSS ADD(started_task_acid) MASTFAC(ZOWE)                          */
+/*                                                                   */
+/* To authorize a user to signon to the FACILITY, issues the         */
+/* following command.                                                */
+/*                                                                   */
+/* TSS ADD(user_acid) FAC(ZOWE)                                      */
+
+/* ................................................................. */
+/* only the last RC is returned, this comment ensures it is a 0      */
```
Comment (OnnoVdT) [inline files/workflows/ZWEWRF02.xml]: also incorporate issue #1044 by replacing this line with
```
/* only the last RC is returned, this command ensures it is a 0      */
PROFILE
```

File: files/workflows/ZWEWRF02.xml [hunk 3]
```diff
@@ -0,0 +1,888 @@
...
+*  general data set protection
+LIST &amp;HLQ.
+RECKEY $&amp;HLQ. ADD(- UID(-) READ(A) EXEC(P))
+RECKEY $&amp;HLQ. ADD(- UID(&amp;SYSPROG.) READ(A) EXEC(A) ALLOC(A) WRITE(A))
+*
+*  show results
+LIST &amp;HLQ.
+*
+* .................................................................
+* only the last RC is returned, this comment ensures it is a 0
```
Comment (OnnoVdT) [inline files/workflows/ZWEWRF02.xml]: also incorporate issue #1044 by replacing this line with
```
/* only the last RC is returned, this command ensures it is a 0      */
PROFILE
```

**Comment by OnnoVdT** [comment]
once this PR is in, I'll help adjsut the build to use the workflow to create the sample JCL, resulting in single source for the ZWESECUR JCL.

File: files/workflows/ZWEYML02.yml [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+#  Group for Zowe administrators
+ADMINGRP=#ZWEADMIN
+
+# STCGROUP
+# Label: STCGROUP
+# Abstract: Group for Zowe started tasks
+# Category: General Security
+# Description:
+#  Group for Zowe started tasks
+STCGROUP=#ZWEADMIN
```
[the remark this answers is not in the document]
Comment (melvl01) [reply files/workflows/ZWEYML02.yml]: done

**Comment by melvl01** [comment]
Fixed, let me know if anything else. Thanks

**Pull request merged**